examples/miscellaneous/plot_fancy_estimator_representation.py

```diff
@@ -0,0 +1,50 @@
+"""
+===========================================
+Displaying estimators and complex pipelines
+===========================================
+
+This example illustrates different ways estimators and pipelines can be
+displayed.
+"""
+
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import OneHotEncoder, StandardScaler
+from sklearn.impute import SimpleImputer
+from sklearn.compose import make_column_transformer
+from sklearn.linear_model import LogisticRegression
+
+
+# %%
+# Compact text representation
+# ---------------------------
+#
+# Estimators will only show the parameters that have been set to non-default
+# values when displayed as a string. This reduces the visual noise and makes it
+# easier to spot what the differences are when comparing instances.
+
+lr = LogisticRegression(penalty="l1")
+print(lr)
+
+# %%
+# Rich HTML representation
+# ------------------------
+# In notebooks estimators and pipelines will use a rich HTML representation.
+# This is particularly useful to summarise the
+# structure of pipelines and other composite estimators, with interactivity to
+# provide detail.  Click on the example image below to expand Pipeline
```

Comment:
It might be worth mentioning clicking on the "horizontal arrow" instead of the image.

Reply:
<img width="501" alt="Screenshot 2022-09-16 at 11 54 46" src="https://user-images.githubusercontent.com/1448859/190612741-685076d7-dc99-4918-af7f-c582d2726878.png">

This is what the image looks like for me, you can click nearly anywhere on that image and something will happen. So I'd vote for leaving the text as is. In addition I took the text nearly verbatim from the "release highlights" section, so I guess someone already thought about this/if it was good enough then it probably is good enough now as well.

WDYT?